feat: disable trade action button after click

shoom3301 opened the pull request:
# Summary

Fixes #195 #1914

#195 doesn't seem relevant anymore. Anyway, I've added `1 sec` loading state after click for all trade form action buttons (`<TradeFormBlankButton />`).

For classic ETH flow I've used the same component `TradeFormBlankButton` and added additional loading state while transaction request is pending.

  # To Test

See #195 #1914

alfetopito reviewed (commented):
I don't think we need this anymore with the new modal, not until we remove the confirmation step.
Also, I'm not a fan of the 1s wait.
I rather to have the button disabled until we get the response from the wallet.

elena-zh reviewed (commented):
Hey @shoom3301 , I really like ETH-flow pending buttons!

However, as for other cases I have concerns: 
1. Wrapping/Unnwrappin/Approve states: a user can press on the back button while a TX is pending, so the button will be clickable again (and it is not in the Pending state)
![back](https://github.com/cowprotocol/cowswap/assets/70885163/1548e5ea-0d08-4828-a918-f668a960fb86)
2. We lost this feature somewhere during refactorings. but the Approve button on the Tokens page also should show 'loading' effect while approving a token
![approve](https://github.com/cowprotocol/cowswap/assets/70885163/0b660ae7-169b-4021-aa09-aca6ac8287dc)
3. When I place an order, and it is in pending signing reques, it is impossible for me to go back (the back button is irresponsive), and that is weird. 2 options here: remove it (if it is not working) OR allow a user to go back and show a loader on the button. 
![not possible (no need)](https://github.com/cowprotocol/cowswap/assets/70885163/eaac47ca-952e-417b-a22b-4d0a6cb7c097)

Thanks!

anxolin commented on apps/cowswap-frontend/src/modules/swap/containers/EthFlow/hooks/useEthFlowActions.ts:
```diff
@@ -88,7 +88,9 @@ export function useEthFlowActions(callbacks: EthFlowActionCallbacks): EthFlowAct
     const expertModeFlow = () => {
-      Promise.all([isApproveNeeded ? approve(false) : undefined, isWrapNeeded ? wrap(false) : undefined])
+      return Promise.all([isApproveNeeded ? approve(false) : undefined, isWrapNeeded ? wrap(false) : undefined]).then(
+        () => void 0
```
nit: a bit hard to read, i think it can be done in more steps instead of a one-liner

shoom3301 replied:
Added `//` to prevent prettier from breaking line break

anxolin reviewed (commented):
Soft approve. just reviewed code. Leaving for Elena the functional test. I will test functionally during the freezing of version

shoom3301 commented:
@elena-zh thank you for the review!

1. This is out of this task scope. I even not sure we can influence on that, because there is no way to reject a request to the wallet from the app side
2. Fixed
3. Fixed

elena-zh reviewed (approved):
I'm approving the current PR, it fixes ETH-flow buttons and makes impossible a double click. 
But I keep opened #187 issue where it would be great to keep buttons (wrap/unwrap/approve) disabled when a TX is sent to the wallet/running.
